src/api.rs

```diff
@@ -73,11 +77,22 @@ struct Stats {
 	running_since: OffsetDateTime
 }
 
+#[derive(Debug, Object)]
+struct BlockInfo {
```

Comment:
I wouldn't necessarily abbreviate "blocklist" as "block".

Reply:
I think the name is fine. Since it the does not especial store info about the blocklist. It is more that the struct store info, why domain is blocked.

Reply:
I'd expect that "block" refers to some block of bytes (in memory, a data format, etc). But maybe that's just me